std_msgs/Header header
OBBDetection[] detections